examples/ex9.cpp

```diff
@@ -437,21 +448,10 @@ int main(int argc, char *argv[])
 FE_Evolution::FE_Evolution(BilinearForm &_M, BilinearForm &_K, const Vector &_b)
    : TimeDependentOperator(_M.Height()), M(_M), K(_K), b(_b), z(_M.Height())
 {
-   bool pa = M.GetAssemblyLevel() == AssemblyLevel::PARTIAL;
-   bool ea = M.GetAssemblyLevel() == AssemblyLevel::ELEMENT;
    Array<int> ess_tdof_list;
-   if (pa || ea)
-   {
-      M_prec = new OperatorJacobiSmoother(M, ess_tdof_list);
-      M_solver.SetOperator(M);
-      dg_solver = NULL;
-   }
-   else
-   {
-      M_prec = new DSmoother(M.SpMat());
-      dg_solver = new DG_Solver(M.SpMat(), K.SpMat(), *M.FESpace());
-      M_solver.SetOperator(M.SpMat());
-   }
+   M_prec = new OperatorJacobiSmoother(M, ess_tdof_list);
+   M_solver.SetOperator(M);
+   dg_solver = NULL;
```

Comment:
This change breaks one of the sample runs, `ex9 -m ../data/amr-quad.mesh -p 1 -r 2 -dt 0.02 -s 13 -tf 9 -no-vis`.

Reply:
Thank you, I'm investigating.

Reply:
@v-dobrev It's fixed.